### PR TITLE
feat: yfm plugins parameter

### PR DESCRIPTION
**Pull request by Kyzyl-ool**

### CODE REVIEW

File: src/data/transformPost.ts [hunk 1]
```diff
@@ -9,9 +9,14 @@ import {Lang} from '../models/locale';
...
  * @param lang - runtime language
  *
+ * @param plugins - YFM plugins list
  * @returns -prepared post
  */
-export const transformPost = (postData: PostData, lang: Lang) => {
+export const transformPost = (
+    postData: PostData,
+    lang: Lang,
+    {plugins}: TransformPostOptions = {},
```
Comment (NikitaCG) [inline src/data/transformPost.ts:18]: I think we need use agrument-object when arguments quantity more than 2, but it is breaking change. I suggest after merge of this PR to make edits in https://github.com/gravity-ui/blog-constructor/pull/137. can you create issue after merge this PR ?

Comment (Kyzyl-ool) [reply src/data/transformPost.ts:18]: Yes,  of course

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@gravity-ui/blog-constructor",
-  "version": "5.7.2",
+  "version": "5.8.0-alpha.0",
```
Comment (NikitaCG) [inline package.json:3]: do we still need this ?

Comment (Kyzyl-ool) [reply package.json:3]: nope, used for testing

**Review by NikitaCG — APPROVED** [review]

**Pull request merged**